modules/custom/az_core/az_core.module

```diff
@@ -384,6 +384,17 @@ function _az_core_check_attribute_form(array &$form, FormStateInterface $form_st
   }
 }
 
+/**
+ * Implements hook_library_info_alter().
+ *
+ * Add style overrides for Chosen.
+ */
+function az_core_library_info_alter(array &$libraries, $module) {
+  if (\Drupal::service('router.admin_context')->isAdminRoute() && $module === 'chosen' && isset($libraries['chosen.claro'])) {
```

Comment:
> Switched to using `hook_library_info_alter()`.

Thanks for making that change!  I haven't tested this but I _believe_ we could omit the `isAdminRoute()` check here because, at least in theory, Chosen is already doing a similar check before it attaches its library to forms.  See:
[ChosenFormRender::preRenderSelect()](https://git.drupalcode.org/project/chosen/-/blob/4.0.x/src/ChosenFormRender.php?ref_type=heads#L32).